notes/test.md

```diff
@@ -1,4 +1,5 @@
 # Tests (links)
 - standard markdown, external - [example a](https://binyam.in)
 - mediawiki-style, internal - [[ bar.md | example b ]]
-- mediawiki-style, internal, doesn't exist - [[ non-existent ]]
\ No newline at end of file
+- mediawiki-style, internal, doesn't exist - [[ non-existent ]]
+- hashtag-style, internal - #tag - [#exampletag](/notes/tag/exampletag)
```

Comment:
Any particular reason we're linking to the tag pages twice?

Reply:
That is a typo on my part. I intended for the URL to be `/notes/tag/` instead of linking to the tag page twice.

> I want a page which lists all notes containing "#art". The page should be /tag/art, or something similar.

This mistake is reproduced in the other links as I referenced the above comment and used `/tag/art` for all URLs. I'll make sure to update it.